.github/workflows/pr_check_client_side_changes.yml

```diff
@@ -27,7 +27,7 @@ jobs:
               repo: context.repo.repo,
               commit_sha: context.sha,
             });
-            const commitHeader = data.message.split('\n')[0];
+            const commitHeader = data.message.split('\n')[0].substring(1); // skip the `#` character
```

Comment:
I'm not sure if this is right, it could be e.g. `'chore: add aria attribute tests (#33184)'` which would end up that the `'c'` gets removed.